service-python/pdfgenerator/src/lib/templates/hypertension-v2.html

```diff
@@ -1,467 +1,489 @@
-        <thead>
-          <tr>
-            <th style="width: 153px !important;">Key terms</th>
-            <th>Observation date</th>
-            <th style="width: 174px !important;">Source</th>
-            <th>VBMS receipt date</th>
-            <th style="width: 120px !important;">Page number(s) / Event ID(s)</th>
-            <th>Document ID</th>
-          </tr>
-        </thead>
-        <tbody>
-          {% for medication in evidence.medications %}
-          <tr>
-            <th>{{ medication.description }}</th>
-            <td {% if medication.dateFormatted == "" %}class="empty" style="line-height: 13px;"{% endif %}>{% if medication.dateFormatted == "" and medication.partialDate == "" %}No associated date{% elif medication.dateFormatted == "" and medication.partialDate != "" %}Non-confident date {{medication.partialDate}}{% else %}{{ medication.dateFormatted }}{% endif %}</td>
-            <td {% if medication.organization == "" and medication.dataSource == "MAS" %}class="empty" {% endif %}>{% if not medication.organization and medication.dataSource == "MAS" %}n.a.{% elif not medication.organization and medication.dataSource == "LH" %} VAMC record {% elif medication.organization and medication.dataSource == "LH" %} VAMC record - {{ medication.organization }}{% else %}{{ medication.organization }}{% endif %}</td>
-            <td {% if medication.receiptDate == "" %}class="empty" {% endif %}>{% if medication.receiptDate == ""%}n.a.{% else %}{{ medication.receiptDate }}{% endif %}</td>
-            <td {% if medication.page == "" %}class="empty" {% endif %}>{% if medication.page == ""%}n.a.{% else %}{{ medication.page }}{% endif %}</td>
-            <td {% if medication.document == "" %}class="empty" {% endif %}>
-              {% if medication.dataSource == "MAS" %}
-                {% if medication.organization == "VAMC Other Output Reports" and medication.document == "" %}
-                  Pending upload to VBMS eFolder
-                {% elif medication.document == "" %}
-                  Pending upload to VBMS eFolder
-                {% else %}
-                  {{ medication.document }}
-                {% endif %}
-              {% else %}
-                {% if medication.document == "" %}
-                  n.a. - data pulled directly from VistA
-                {% else %}
-                  {{ medication.document }}
-                {% endif %}
+    <div class="section" style="padding-left: 40px; padding-top: 20px; padding-bottom: 40px;">
+      <div class="row">
+        <div class="col">
+          <h2 class="m-0 mb-13"><b>Schedular evidence for hypertension</b></h2>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col">
+          <table id="schedular-evidence" class="table table-bordered">
+            <thead>
+              <tr>
+                <th style="width: 153px !important;">Key terms</th>
+                <th>Observation date</th>
```

Comment:
Looks too wide compared to figma document. maybe we should set a standard width for 'Observation date' and 'VBMS receipt date' columns

Reply:
Maybe `90 px`